test/helper.rb

```diff
@@ -23,7 +23,8 @@
 
 ActiveRecord::Base.logger = Logger.new(__dir__ + "/test.log")
 ActiveSupport.test_order = :sorted
-ActiveSupport::Deprecation.behavior = :raise
+ActiveSupport::Deprecation.behavior = :silence
```

Comment:
How do folks feel about this? We get a TON of deprecation warnings otherwise.

Reply:
That noise is good, right? It's something for us to action. I'm afraid that if we silence them they'll get forgotten about.

Reply:
What kind of warnings? We also get warnings from https://github.com/zendesk/active_record_shards/blob/693c4eaf9250603215ee41f6553b8c4df6a2f44a/Rakefile#L11, right?

Reply:
We get around 6+ warnings per test:

```
DEPRECATION WARNING: Using `"primary"` as a `connection_specification_name` is deprecated and will be removed in Rails 7.0.0. Please use `ActiveRecord::Base`. (called from assert_using_database at /Users/nony.duDEPRECATION WARNING: [] is deprecated and will be removed from Rails 7.0 (Use configs_for) (called from connection_specification_name at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher-6-0.rb:8)
DEPRECATION WARNING: [] is deprecated and will be removed from Rails 7.0 (Use configs_for) (called from config_for_env at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher.rb:147)
DEPRECATION WARNING: [] is deprecated and will be removed from Rails 7.0 (Use configs_for) (called from connection_specification_name at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher-6-0.rb:8)
DEPRECATION WARNING: Using `"primary"` as a `connection_specification_name` is deprecated and will be removed in Rails 7.0.0. Please use `ActiveRecord::Base`. (called from ensure_shard_connection at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher-6-0.rb:26)
DEPRECATION WARNING: Using `"primary"` as a `connection_specification_name` is deprecated and will be removed in Rails 7.0.0. Please use `ActiveRecord::Base`. (called from ensure_shard_connection at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher-6-0.rb:26)
DEPRECATION WARNING: Using `"primary"` as a `connection_specification_name` is deprecated and will be removed in Rails 7.0.0. Please use `ActiveRecord::Base`. (called from ensure_shard_connection at /Users/nony.dutton/Code/zendesk/active_record_shards/lib/active_record_shards/connection_switcher-6-0.rb:26)

```

Reply:
How about e.g. ` ActiveSupport::Deprecation.behavior = :silence if ActiveRecord::VERSION::MAJOR == 6 && ActiveRecord::VERSION::MINOR == 1`?

Reply:
Done!